app/routes/_public.undercut.tsx

```
@@ -1,339 +1,336 @@
+                  ...prev,
+                  add_ids: value === 'selected' ? prev.add_ids : [],
+                  ignore_ids: value === 'all' ? prev.ignore_ids : []
+                }))
+              }}
+              defaultChecked={alertType}
+            />
+
+            <div className="flex flex-col gap-4 mt-4">
+              <button
+                className="w-full py-2 px-4 text-sm bg-gray-100 border-gray-300 rounded text-left dark:bg-gray-600 dark:text-gray-200 dark:border-gray-600"
+                onClick={() => setModal({ type: 'retainers', open: true })}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)